src/main/java/com/notably/logic/parser/NotablyParser.java

```diff
@@ -10,6 +10,7 @@
 import com.notably.logic.commands.DeleteCommand;
 import com.notably.logic.commands.EditCommand;
 import com.notably.logic.commands.ExitCommand;
+import com.notably.logic.commands.HelpCommand;
```

Comment:
btw for readability, can you add a newline between line 38 and 39?

Reply:
Im not sure why, but when i add a new line in between line 38 and 39. It counts as trailing whitespace.